app/views/memberships/_confirm_change_permissions.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :page_title, "Change permissions - #{yield :page_title}", flush: true %>
+
+<% if @membership.can_manage_team_as_admin_with_limitations? %>
+  <%= render "memberships/cannot_change_permissions" %>
```

Comment:
I think you probably have to put a conditional on the `update` method in the controller as well and not only in the view. Otherwise you could in theory bypass this by sending a custom update request.Maybe also add a request spec to verify that